x-pack/plugins/cloud_security_posture/public/components/vulnerability_badges.tsx

```diff
@@ -39,55 +38,32 @@ export const CVSScoreBadge = ({ score, color, version }: CVSScoreBadgeProps) =>
         }
       `}
     >
-      <EuiTextColor color="ghost">{score}</EuiTextColor>
-      <hr
-        css={css`
-          width: 1px;
-          border: 0 none;
-          background-color: rgba(255, 255, 255, 0.2);
-          margin: 0px 6px;
-        `}
-      />
-      <EuiTextColor color="ghost">{versionDisplay}</EuiTextColor>
+      {versionDisplay && (
+        <>
+          <EuiTextColor color="ghost">{score}</EuiTextColor>
+          <hr
+            css={css`
+              width: 1px;
+              border: 0 none;
+              background-color: rgba(255, 255, 255, 0.2);
+              margin: 0px 6px;
+            `}
+          />
+          <EuiTextColor color="ghost">{versionDisplay}</EuiTextColor>
+        </>
+      )}
     </EuiBadge>
   );
 };
 
-export const SeverityStatusBadge = ({ color, status }: SeverityStatusBadgeProps) => {
-  const xxsFontSize = useEuiFontSize('xxs').fontSize;
-  const { euiTheme } = useEuiTheme();
+export const SeverityStatusBadge = ({ score, status }: SeverityStatusBadgeProps) => {
+  const color = getCvsScoreColor(score);
 
   return (
-    <EuiBadge
-      color={'hollow'}
-      css={css`
-        border: none;
-        .euiBadge__text {
-          display: flex;
-          font-weight: 400;
-          font-size: ${xxsFontSize};
-          align-items: center;
-        }
-      `}
-    >
-      {color && (
-        <span
-          css={css`
-            border-radius: 50%;
-            background-color: ${color};
-            width: ${euiTheme.size.s};
-            height: ${euiTheme.size.s};
-            margin-right: 6px;
-          `}
-        />
-      )}
-      <EuiText>
-        {i18n.translate('xpack.csp.vulnerabilityBadges.severityStatusBadge', {
-          defaultMessage: '{status}',
-          values: { status },
-        })}
-      </EuiText>
-    </EuiBadge>
+    <>
+      <EuiIcon type="dot" color={color} />
+      {status}
```

Comment:
Shouldn't we translate the status? Also, I didn't add this but can we capitalize the status? Status shows in all caps which differs from mockups.

Reply:
It's not a good idea to translate or change the casing here, because if the user tries to query `finding.vulnerability.severity : High` instead of `finding.vulnerability.severity : HIGH` for example, it will get no results, the same applies to translations, so it might be confusing.

Reply:
I agree with @opauloh , lets open another issue to fix that value.
Currently, for the `beta` we can move on the way it is

Reply:
Ahh i see! okay, make sense.